services/user-svc/internal/hwkc/client.go

```diff
@@ -16,6 +16,16 @@ import (
 	"strconv"
 )
 
+type IClient interface {
```

Comment:
in a comment mention implementors, duck-typing is a pain to resolve statically without tooling (e.g. on github)

Reply:
yup